doc/stdlib/math.rst

```diff
@@ -773,6 +1257,7 @@ Mathematical Functions
       .. doctest::
```

Comment:
Stray ``.. doctest::``. There's a few others as well, but they seem to have been around for a while, see https://github.com/JuliaLang/julia/blob/103f7a3b47f5a476c8d1297a466a1e3d6ad6263c/base/docs/helpdb.jl#L4744.